From dd2efa50d197e2205acd8edbf29aba717d71bdbb Mon Sep 17 00:00:00 2001
From: Yang Yanchao <yangyanchao6@huawei.com>
Date: Tue, 4 Jan 2022 19:55:28 +0800
Subject: [PATCH] testsuit: delete check-installed-headers-c and check-installed-headers-cxx

check-installed-headers-c and check-installed-headers-cxx checked at CI
and can be deleted during building

---
 Makefile | 4 ++--
 Rules    | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index f98d5a9e..83200a75 100644
--- a/Makefile
+++ b/Makefile
@@ -532,7 +532,7 @@ $(objpfx)check-local-headers.out: scripts/check-local-headers.sh
 
 ifneq "$(headers)" ""
 # Special test of all the installed headers in this directory.
-tests-special += $(objpfx)check-installed-headers-c.out
+# tests-special += $(objpfx)check-installed-headers-c.out
 libof-check-installed-headers-c := testsuite
 $(objpfx)check-installed-headers-c.out: \
     scripts/check-installed-headers.sh $(headers)
@@ -542,7 +541,7 @@ $(objpfx)check-installed-headers-c.out: \
 	$(evaluate-test)
 
 ifneq "$(CXX)" ""
-tests-special += $(objpfx)check-installed-headers-cxx.out
+# tests-special += $(objpfx)check-installed-headers-cxx.out
 libof-check-installed-headers-cxx := testsuite
 $(objpfx)check-installed-headers-cxx.out: \
     scripts/check-installed-headers.sh $(headers)
diff --git a/Rules b/Rules
index b1137afe..ca29abd7 100644
--- a/Rules
+++ b/Rules
@@ -84,7 +84,7 @@ common-generated += dummy.o dummy.c
 ifneq "$(headers)" ""
 # Test that all of the headers installed by this directory can be compiled
 # in isolation.
-tests-special += $(objpfx)check-installed-headers-c.out
+# tests-special += $(objpfx)check-installed-headers-c.out
 libof-check-installed-headers-c := testsuite
 $(objpfx)check-installed-headers-c.out: \
     $(..)scripts/check-installed-headers.sh $(headers)
@@ -96,7 +95,7 @@ $(objpfx)check-installed-headers-c.out: \
 ifneq "$(CXX)" ""
 # If a C++ compiler is available, also test that they can be compiled
 # in isolation as C++.
-tests-special += $(objpfx)check-installed-headers-cxx.out
+# tests-special += $(objpfx)check-installed-headers-cxx.out
 libof-check-installed-headers-cxx := testsuite
 $(objpfx)check-installed-headers-cxx.out: \
     $(..)scripts/check-installed-headers.sh $(headers)
-- 
2.27.0

